src/celeritas/Types.hh

```diff
@@ -160,6 +160,17 @@ enum class MscStepLimitAlgorithm
     size_,
 };
 
+//---------------------------------------------------------------------------//
+//! Nuclear form factor model for Coulomb scattering
+enum class NuclearFormFactorType
```

Comment:
I moved the enum here because we're now also using it in the import data structs, a lot of our other enums live here, and we shouldn't need additional changes for the SWIG build (#1159), but there wouldn't be any problem with leaving in the wentzel data. I think as long as the form factor calculations are only used in the wentzel distribution we can leave them there.